src/commands/devchain.js

```diff
@@ -88,8 +90,10 @@ exports.task = async function ({ port = 8545, reset = false, showAccounts = 2 })
 }
 
 exports.printAccounts = (reporter, privateKeys) => {
+  const firstAccountComment = '(this account is used to deploy DAOs, it has more permissions)'
+
   const formattedAccounts = privateKeys.map(({ address, key }, i) => 
-    chalk.bold(`Address #${i + 1}: ${address}\nPrivate key: `) + key
+    chalk.bold(`Address #${i + 1}:  ${address} ${ i == 0 ? firstAccountComment : ''}\nPrivate key: `) + key
```

Comment:
Seems like there's an extra space now at the start of the address?

Reply:
Vanity feature: addresses and private keys are horizontally aligned to the left now (rather than unaligned by one char).